cmd/cli/metrics_disable.go

```diff
@@ -64,10 +64,21 @@ func (cmd *metricsDisableCmd) run() error {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 
-		if _, err := cmd.clientSet.CoreV1().Namespaces().Get(ctx, ns, metav1.GetOptions{}); err != nil {
+		namespace, err := cmd.clientSet.CoreV1().Namespaces().Get(ctx, ns, metav1.GetOptions{})
+		if err != nil {
 			return errors.Errorf("Failed to retrieve namespace [%s]: %v", ns, err)
 		}
 
+		// Check if the namespace belongs to a mesh, if not return an error
+		monitored, err := isMonitoredNamespace(*namespace, getMeshNames(cmd.clientSet))
```

Comment:
I imagine using `metav1.GetOptions{}` to match meshnames (if even entirely possible) will just complicate your life

Reply:
You mean to list namespaces whose labels match against a list of mesh names?